server/src/test/java/org/elasticsearch/index/mapper/DateFieldMapperTests.java

```diff
@@ -734,7 +734,7 @@ public void testLegacyDateFormatName() {
             null,
             mock(ScriptService.class),
             true,
-            Version.CURRENT.minimumIndexCompatibilityVersion() // BWC compatible index, e.g 7.x
+            Version.fromId(7_99_99_99) // BWC compatible index, e.g 7.x
```

Comment:
One last small comment :) It's unusual to use `Version.fromId` in tests. We'd usually choose a random version from a range, or even a fixed version like `Version.V_7_17_0`.